workers/migrations/0025-stacktemplate-index.js

```diff
@@ -0,0 +1,20 @@
+
+var mongodb = require('mongodb');
+
+exports.up = function(db, next){
+  var index = {
+    "name": "groupName_originId_slug",
+    "key": {
+      "slug": 1,
```

Comment:
with the current code base it seems having slug at the end could be better use of the index - since there are other queries that uses originId and group. as you know property order matters in mongo indexes